### PR TITLE
Save enriched ads to a Pickle file, create index from that file

### PR DESCRIPTION
**Pull request by oskardrenske-ith**
Save enriched ads to a Pickle file so that they can be imported from that file in any Elastic

### CODE REVIEW

**Review by halge — APPROVED** [review]

File: importers/platsannons/main.py [hunk 1]
```diff
@@ -126,10 +129,19 @@ def _load_and_process_ads(ad_ids, es_index, es_index_deleted):
...
 
     return doc_counter
 
 
+def save_enriched_ads():
+    """
+    saves enriched ads to a Pickle file that can be used when creating an index from a known state
+    """
+    if settings.SAVE_ENRICHED_ADS:
+        save_enriched_ads_to_file(enriched_ads_to_save)
```
Comment (halge) [inline importers/platsannons/main.py:142]: enriched_ads_to_save: will it be accessible from another func? it is not global.

File: importers/settings.py [hunk 2]
```diff
@@ -57,3 +57,5 @@
 #taxonomy replaced terms
 TAXONOMY_2_URL = os.getenv('TAXONOMY_2_URL')
 TAXONOMY_2_API_KEY = os.getenv('TAXONOMY_2_API_KEY')
+
+SAVE_ENRICHED_ADS = False
```
Comment (halge) [inline importers/settings.py:61]: use os.getenv with default.

**Review by halge — CHANGES_REQUESTED** [review]

File: importers/platsannons/main.py (hunk 1 above)
Comment (oskardrenske-ith) [reply importers/platsannons/main.py:142]: works without 'global' keyword, but I added it for readability

Comment (oskardrenske-ith) [reply importers/platsannons/main.py:142]: yes it works. I added 'global' keyword where ads are saved to 'enriched_ads_to_save' list for readability

**Review by halge — APPROVED** [review]

**Review by lina1101 — APPROVED** [review]

**Pull request merged**